src/main/main.ts

```diff
@@ -1,9 +1,11 @@
 import { PROJECT_ID } from "./basecamp";
-import { generateIdForRow, getBasecampTodoRequestsForRow, getId, hasId, saveRow } from "./row";
+import { RowBasecampMappingMissingError } from "./error/rowBasecampMappingMissingError";
+import { getRoleTodoIdMap } from "./role";
+import { generateIdForRow, getBasecampTodoRequestsForRow, getId, getRowBasecampMapping, hasChanged, hasId, saveRow } from "./row";
 import { getEventRowsFromSpreadsheet } from "./scan";
-import { createTodo, TODOLIST_ID } from "./todos";
+import { createNewTodos, createTodo, createTodosForNewRoles, deleteObsoleteTodos, deleteTodo, deleteTodos, TODOLIST_ID, updateTodo, updateTodosForSurvivingRoles } from "./todos";
```

Comment:
There are some unused imports here after the refactoring

Reply:
Removed unused imports